src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/clients/pcq/PcqClientTest.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.hmcts.reform.blobrouter.clients.pcq;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import feign.FeignException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.blobrouter.clients.response.SasTokenResponse;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.notFound;
+import static com.github.tomakehurst.wiremock.client.WireMock.okJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.unauthorized;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+
+@AutoConfigureWireMock(port = 0)
+@ActiveProfiles("integration-test")
+@SpringBootTest(properties = "pcq-backend-api-url=http://localhost:${wiremock.server.port}")
+public class PcqClientTest {
+
+    @Autowired
+    private PcqClient client;
+
+    @Test
+    public void should_return_sas_token_when_service_authorization_header_is_valid() {
+
+        var expected = new SasTokenResponse("pcq-sas-token");
+
+        // given
+        stubFor(get("/pcq/backend/token/bulkscan").willReturn(okJson("{\"sas_token\":\"pcq-sas-token\"}")));
```

Comment:
it would be good to specify for which headers this will actually work

Reply:
Updated the test